modules/Bio/EnsEMBL/Compara/Utils/FlatFile.pm

```diff
@@ -64,19 +64,20 @@ our @EXPORT_OK;
 =cut
 
 sub map_row_to_header {
-    my ($line, $header) = @_;
+    my ($line, $header, $delimiter_pattern) = @_;
+    $delimiter_pattern //= '\s+';
     
     chomp $line;
     chomp $header;
-    my @cols      = split(/\s+/, $line);
+    my @cols      = split(/$delimiter_pattern/, $line);
```

Comment:
Should the LIMIT be set to -1 here too?
```suggestion
    my @cols      = split(/$delimiter_pattern/, $line, -1);
```

Reply:
As the given `line` values should not contain whitespace, we should not preserve it. Isn't? @twalsh-ebi